docs/services.rst

```diff
@@ -111,6 +115,11 @@ is ``'Caller-Destination-Number'``) and ``kwargs``.
 The ``pattern`` must be matched against the ``field`` *event header* in order for
 the *route* to be called with ``kwargs`` (i.e. ``reject_international(**kwargs)``).
 
+
+You can run this app directly using ``switchy serve``::
+
+    switchy serve freeswitch1.net freeswitch2.net --app ./dialplan.py:router
+
```

Comment:
@k4ml no actually it **should** be `router` because that is the instance of `Router` which has been created in that file which has already had routes registered.

So the part after the `:` is the attribute name that will be looked up as the `switchy` app.